inspire_matcher/utils.py

```diff
@@ -92,3 +93,10 @@ def get_tokenized_title(title):
 
     """
     return set(title.lower().split())
+
+
+def get_prefixed_index(index_name, index_prefix=None):
+    prefix = index_prefix or current_app.config.get("SEARCH_INDEX_PREFIX")
+    if prefix and prefix not in index_name:
```

Comment:
* If you don't mutate the config you receive, you can assume you always receive a non-prefixed index, and you don't need to test if the prefix is already in the index name.
* If you also do
```suggestion
    prefix = current_app.config.get("SEARCH_INDEX_PREFIX") or ""
```
you can avoid the `if` prefix, as `"" + index_name == index_name`.